src/chart/graph/component/GraphEntityInspectionTable.tsx

```diff
@@ -91,7 +94,8 @@ export const GraphEntityInspectionTable = ({
                     {key}
                   </TableCell>
                   <TableCell align={'left'} style={{ color: tableTextColor }}>
-                    <ShowMoreText lines={2}>{formatProperty(entity?.properties[key])}</ShowMoreText>
```

Comment:
Actually keep this line instead of the other one, cause e parse toString inside the formatProperty method now